### PR TITLE
Successors Assets

### PR DESCRIPTION
**Pull request by Daeridanii1**
This should be everything that wasn't just destructively edited

### CODE REVIEW

File: effects/successor pulse laser hit.xcf
Comment (tibetiroka) [inline effects/successor pulse laser hit.xcf:1]: Are there no assets for the overcharged laser hit?

File: outfits/tiny successor engines.xcf
Comment (tibetiroka) [inline outfits/tiny successor engines.xcf:1]: It's called `successor engines tiny` in the main PR. Same for the blend file.

File: projectiles/sjeja puncture rail.xcf
Comment (tibetiroka) [inline projectiles/sjeja puncture rail.xcf:1]: This isn't in the main PR.

File: ships/kvelqde-khora thumb.xcf
Comment (tibetiroka) [inline ships/kvelqde-khora thumb.xcf:1]: Thumbnails should be in `thumbs`, and with the correct name. Also, should the other ships have them as well, or does this have something special going on?

File: ships/nnaatej1.blend
Comment (tibetiroka) [inline ships/nnaatej1.blend:1]: What is the purpose of this file?

File: thumbs/aaulqra thumb.xcf
Comment (tibetiroka) [inline thumbs/aaulqra thumb.xcf:1]: These files should only have the ship name (without `thumb`).

**Review by tibetiroka — COMMENTED** [review]
The hardpoints seem to be missing.

Also, the files for `successor armor keystone` are missing.

Does the void scar have an xcf file?

**Comment by Daeridanii1** [comment]
Hardpoints are a separate scene in the relevant turret .blend files.

File: effects/successor pulse laser hit.xcf
Comment (Daeridanii1) [reply effects/successor pulse laser hit.xcf:1]: There are, see `effects/overcharged pulse laser hit.xcf`

Comment (tibetiroka) [reply effects/successor pulse laser hit.xcf:1]: Ah, my bad.

**Comment by Daeridanii1** [comment]
I destructively edited `void scar.png`, so there's no .xcf with more data. I can create one if it's needed, but it will just be the @2x image in a .xcf.

**Comment by tibetiroka** [comment]
Nah, I was just hoping there was a better source available.

**Review by tibetiroka — APPROVED** [review]

**Pull request merged**